### ISSUE TITLE
Reversible microbial colonization of germ-free mice reveals the dynamics of IgA immune responses.

### ISSUE THREAD

**Issue by pschloss** [SchlossLab/march_madness_2015#20]
Reversible microbial colonization of germ-free mice reveals the dynamics of IgA immune responses.
Hapfelmeier S, Lawson MA, Slack E, Kirundi JK, Stoel M, Heikenwalder M, Cahenzli J, Velykoredko Y, Balmer ML, Endt K, Geuking MB, Curtiss R 3rd, McCoy KD, Macpherson AJ.
Science. 2010 Jun 25;328(5986):1705-9. doi: 10.1126/science.1188454.
http://www.ncbi.nlm.nih.gov/pubmed/20576892

**Comment by mjenior** [comment]
In this paper from Andrew MacPherson's group, they sought to examine the role of reversibly associating live bacteria with germ-free mice on the development of immunity, specifically lasting mucosal IgA responses.  They hypothesized that if they could temporarily colonize germ-free animals with live bacteria, they would see lasting effects on host immunity that are not present when animals are given killed bacteria.   To accomplish this, they first engineered a double auxotrophic deletion mutant of E. coli K-12 for both meso-diaminopimelic acid and D-alanine biosynthesis.  The double auxotrophies ensure that it would be highly unlikely for the bacteria to revert to wildtype and permanently colonize the GI tract of the monoassociated animals.  When given to the mice, these bacteria could not divide or persist in the GI tract but were easily recovered/cultured from the feces.  They used this system to find that the animals develop specific a humoral response to the species given that does not bind general bacterial antigens, only those from the specific strain given first.  However, as new species were introduced the repertoire of IgA shifted to include antibodies specific to the new group as well.  Any of these responses was also dose dependent as they saw no new mucosal IgA below 10^8 bacteria given per dose.  Most importantly, none of this immune development was seen when germ-free mice were given equivalent quantities of heat-killed bacteria.  These immune responses were also consistently detectable over 16 weeks after bacterial removal, suggesting long lasting immunological memory.  Interestingly, following repeated doses of bacteria they found that the immune response to the commensal species was additive.  This is in contrast to vaccination where repeated exposure to an antigen results in synergistic and exponential increases in specific antibody titers.  This may be due to these exposures being limited to the GI tract as vaccinations are systemic and yield an equally systemic, primarily IgG, response.  This paper is important because it demonstrates how gut IgA repetoires continuously adapt to protect from the current bacterial community and how the immune response could adapt to changing community structures over time.

TL;DR – Colonization with enough LIVE bacteria result in development of specific, lasting mucosal antibodies even when the species is now longer present.

Overall: 6
Importance: 8
Innovation: 10
Who should read:  more general knowledge for all of us 
Tags: gnotobiotic, immunity, auxotroph

**Comment by pschloss** [comment]
The manuscript by Hapfelmeier and colleagues uses a highly innovative and elegant method to probe the memory of the gut immune system. I admit that much of the manuscript was lost on me as it is very immunological, but the take home messages are pretty clear and the overall method should be of great interest to microbial ecologists. Basically, they engineered a auxotrophic strain of E. coli that could not grow in a germ-free mouse. When they gavaged it into mice it quickly (hours) got washed out of the system. Thus they call it a "reversible" colonization system. To quickly summarize the immunological points of the study they include:

1. you can get high levels of IgA even without permenant colonization
2. there is a certain number of bacteria that are needed to trigger the IgA response
3. the IgA response is specific to whatever is most abundant at the time and has little memory

They do make a claim, that I found confusing. They stated that "...E. coli HA107, which cannot divide and persist in vivo..." Yet, when I look at Figure 1A, I see it increasing in abundance between 3 and 9 hours. Wouldn't this suggest growth? Furthermore, that the response was not seen in heat-killed bacteria would seem to suggest that the bacteria actually doing something is important to the response.

I was also surprised that the authors did not include two controls. First, I think they could have complemented the auxotrophy by providing meso-diaminopimelic acid in the diet or water. Second, at a minimum, they could have put the missing genes on a plasmid and shown that the organism persists in the host. Third, this is definitely more my question than theirs, but it would have been really cool to co-innoculate with another auxotroph and see whether they would complement each other in the gut. I think that this third point is really where the potential lies for this type of approach.

Overall: 8
Importance: 4
Innovation: 8
Who should read: Jenior, Alyx
Tags: gut immunology, germ-free models

**Comment by nbaxter13** [comment]
Hapfelmeier et al. investigate the dynamics of IgA production in response to gut microbiota, using “reversible colonization” of germ-free mice.  They do so by engineering and E. coli K12 auxotroph that is unable to stably colonize the mouse gut.  They give germ-free mice heavy doses of bacteria, which stimulate an IgA response and are then washed out.  They show that this pulse of live bacteria is sufficient to elicit a memory response (though relatively short-lived) and that increasing the number of doses of bacteria  had an additive effect on E. coli-specific IgA titers.  Interestingly, heat-killed E. coli elicited a reduced response, suggesting the immune system is able to distinguish live bacteria from dead bacteria (see Denise Bronner’s thesis work).  Alternatively, as Pat suggested, the E. coli could actually be growing in the gut and producing something that stimulates the immune system.  

Like Pat said, they needed to supplement the diet of the mouse so that the E. coli could stably colonize.  That would have been much better than giving repeated heavy doses of something that doesn’t colonize.  Then again, that’s how most probiotics work.   Their method is innovative but has room for improvement.  Their findings are interesting, but far from surprising.   IgA production likely plays an important role in shaping the microbiome, especially early in life (see Schloss et al 2012 in Gut Microbes).  More studies like this one, but with multiple types of bacteria and ones that can actually colonize, would help better understand how the microbiome shapes long-term immunity and vice versa. 

Overall: 8
Importance: 6
Innovation: 8
Who should read: @nbaxter13 @mjenior @alyxwithay
Tags: immunity, probiotics, germ-free
